docs/modules/ROOT/pages/how-to-guides/testing_applications/proc_creating_custom_test.adoc

```diff
@@ -91,6 +91,16 @@ spec:
 . Add your new custom test as an integration test in {ProductName}.
 .. For additional instructions on adding an integration test, see this document: xref:how-to-guides/testing_applications/proc_adding_an_integration_test.adoc[Adding an integration test].
 
+----
+In the integration test created above, integration-service adds some Params/Labels automatically to the integration PipelineRun
+
+1. A param containing the Snapshot as a json string
+2. A param containing the DeploymentTarget connection details and Environment name
+3. A Label for type of PipelineRun, i.e. "test"
+4. A Label for optional flag of IntegrationTestScenario 
+5. Labels for the name of application, component and IntegrationTestScenario
```

Comment:
Shall we write explicitly names of params and labels?

Also examples would be helpful

Reply:
What examples? Do you mean the current example for the IntegrationPipelineRun is too simple and replace it with a new one?

Reply:
yes, like complete value what user can see in SNAPSHOT param

Reply:
@MartinBasti , do you mean add an example value for SNAPSHOT like this one ? {"application":"application-sample","components":[{"name":"component-sample","containerImage":"quay.io/redhat-appstudio/sample-image@sha256:34fb1c72bc976937d0a59849160746cec12a8c925f31e74dc9f3b922f74da0e9","source":{"git":{"url":"git@github.com:redhat-appstudio/integration-service.git\n","revision":"sha256:3703843e80f39188c060a6ed0f524519a682be1e7141fee29b878b1973e9783c"}}}],"artifacts":{}}

Reply:
It's so long that I don't want to add it.

Reply:
yes

that's why I wanted to make it a separate textual section, which gives us more space

if SNAPSHOT JSON cannot be added, we should at least link it to SNAPSHOT definition, so use can easily find which fields are available

Reply:
Added the link to Snapshot definition as you suggested. Thanks